scripts/mock_sds_endpoint.py

```diff
@@ -1,21 +1,32 @@
 import json
 
+import yaml
 from flask import Flask, Response, request
+from sdc.crypto.jwe_helper import JWEHelper
+from sdc.crypto.key_store import KeyStore
+
+from app.keys import KEY_PURPOSE_SDS
 
 app = Flask(__name__)
+with open("dev-keys.yml", encoding="UTF-8") as keys_file:
```

Comment:
should this use environment variable instead of the filename directly? I guess its a mock endpoint so it doesn't really matter